dashboard/test/ui/step_definitions/top_instructions.rb

```diff
@@ -10,10 +10,9 @@
 # it all the way through, by clicking on play, waiting for "play" to
 # turn into "pause", and then waiting for "pause" to turn back into
 # "play"
-Then /^I listen to the entirety of the "(\d*)"th inline audio element$/ do |n|
+Then /^I listen to the (\d+)(?:st|nd|rd|th) inline audio element$/ do |n|
   steps <<-STEPS
     Then I click selector ".csf-top-instructions .inline-audio:eq(#{n}) button"
-    Then I wait until element ".csf-top-instructions .inline-audio:eq(#{n}) button img" is an img with src "/blockly/media/common_images/pause.png"
-    Then I wait until element ".csf-top-instructions .inline-audio:eq(#{n}) button img" is an img with src "/blockly/media/common_images/play.png"
+    And element ".csf-top-instructions .inline-audio:eq(#{n}) button img" is visible
```

Comment:
should probably add a "And I wait 2 seconds" above this line, just to make sure it actually has time to error out if it's gonna.